### DISCUSSION TITLE
A few suggestions to get closer to Sidebery on Firefox

### DISCUSSION THREAD

**Discussion by Elaws** [oxdc/obsidian-vertical-tabs#80] [Ideas]
Great plugin, well done !

I wish it could include some features to get closer to [Sidebery](https://github.com/mbnuqw/sidebery) on Firefox (see also related [feature request](https://forum.obsidian.md/t/sidebery-like-tab-navigation-for-obsidian/59456) by @Pyrodusk ) :

- Possibility to select several tabs and group them with a custom name.
- Multi-level subtree : middle-click on a link to open as a child of current tab.
- Drag a tab to a parent tab.
- Vertical workspaces.

![image](https://github.com/user-attachments/assets/f83dee9e-c47d-4266-8399-fb66f40a52ab)

Sidebery could probably be our gold standard for better navigation in Obsidian.

Looking forward to hearing your thoughts about it !

**Comment by oxdc** [comment]
Thank you! I notice that you have posted the same feature request in multiple places. I'd appreciate your enthusiasm. Sorry for my late reply, I've been busy these days. I'll read and take a note of every request or comment, no matter where you posted it.

The good news is that all these features you are hoping for are already in my plan. But I have a rather tight schedule and a huge refactoring is still under way. There would be some time before releasing. Please stay tuned 🎉

Also some notes:

The final implementation would NOT be an exact clone of Sidebery. Sidebery's workflow is optimized for web browser environment, while _Vertical Tabs_ must adapt to and respect Obsidian's unique interface design. My intention is to make something more intuitive and manageable than Sidebery, and also fine tuned for note taking.

When surfing the Internet, it's easy to fall into the rabbit hole with hundreds of tabs open and losing track of "why would I open this in the first place". In this use case, Sidebery provides an almost perfect solution.

When you are writing notes, or skipping the notes and looking for insights, there is little chance that you would open _that_ much of tabs. Even so, you might hardly get anything deep from overloaded information.

In Obsidian, the need of managing tons of tabs happens most likely when you are searching and skipping. You'd better close all the notes when the work is done. If too many notes are permanently open, _Vertical Tabs_ would become a file browser.

The best is not to spend much time organizing your _ephemeral_ tabs into a delicate tree. This situation is pretty much like what Zettelkasten has been advertising for. You don't want to use up your brain, your precious attention to these details. And sometimes it's hard to find the right folder to put your notes. The same applies for tab management. You may find that a tab is suitable in multiple places.

My design is intended to provide the same functionality as Sidebery and supplement with a simpler but better solution, optimized for Obsidian.

I'll keep this issue open for tracking the progress.

**Comment by Elaws** [comment]
Thanks for your reply @oxdc ! 

While I completely agree that having 100+ tabs opened on Obsidian may not be the most productive, I can definitely see a usecase for a few tabs laid out in a multi-level tree structure, and in different workspaces (also laid out vertically next to the tab tree) as in Sidebery. 

Looking forward to hearing news about these features !

Have a great day

**Comment by oxdc** [comment]
> I can definitely see a usecase for a few tabs laid out in a multi-level tree structure, and in different workspaces

Yes, this would be included!

**Comment by obsidiest** [comment]
+1 for multi-level trees.
